saleor/core/utils/__init__.py

```diff
@@ -153,3 +153,18 @@ def prepare_unique_attribute_value_slug(attribute: "Attribute", slug: str):
         "slug", flat=True
     )
     return prepare_unique_slug(slug, value_slugs)
+
+
+def queryset_in_batches(queryset, batch_size):
+    """Slice a queryset into batches.
+
+    Input queryset should be sorted be pk.
```

Comment:
[Question] Maybe instead of writing that the query should be sorted, we can make this sort here?